main.go

```diff
@@ -104,44 +100,50 @@ func main() {
 		Client:           mgr.GetClient(),
 		Log:              ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
 		Scheme:           mgr.GetScheme(),
-		ConfigSecretName: envVars.configName,
+		ConfigSecretName: EnvVars[addonParamsSecretEnvVarName],
```

Comment:
`ConfigSecretName` - Please see my last review comments regarding naming.
I can see the naming changes in the tests code but not on the actual controller code